homeassistant/components/doorbird/config_flow.py

```diff
@@ -28,6 +35,9 @@ def _schema_with_defaults(host=None, name=None):
             vol.Required(CONF_HOST, default=host): str,
             vol.Required(CONF_USERNAME): str,
             vol.Required(CONF_PASSWORD): str,
+            vol.Optional(CONF_PORT, default=80): vol.Coerce(int),
+            vol.Optional(CONF_SSL, default=False): bool,
+            vol.Optional(CONF_MJPEG_VIDEO, default=False): bool,
```

Comment:
This looks like it belongs in an option flow since it can be changed later after the integration is set up

Reply:
I initially did intend to put it there, but it does require some significant changes to the rest of the code to make changing it after the fact possible. In reality, I'm not sure there would ever be a need to change this value; it's real value is if you are behind a proxy, you can't use the rtsp stream at all.

Reply:
Why does it require significant changes to the rest of the code to expose these settings as options as well as or instead of adding them in the config flow?
Normally this is handled by letting the config entry's update listener trigger a reload of the config entry, in that way you don't need to do anything complex to have the new settings applied when the config entry is already setup.

Reply:
Will that recreate the platform entities? That's the part that I imagined would take some work. I am admittedly still figuring out my way around this code.

The only setting that makes sense at all as a post setup option is the mjpeg option. Without the other settings, you won't even be able to connect, so you can't make it past the initial config flow.